docs/faq.rst

```diff
@@ -0,0 +1,114 @@
+.. _faq:
+
+Frequently asked questions
+==========================
+
+General fairness questions
+--------------------------
+
+Where can I learn more about fairness in machine learning?
+    For a high level view of the subject, please see the :ref:`further reading <further_reading>`
+    section, where we provide links to a variety of books and papers.
+    In our :ref:`user guide <user_guide>` we also have links to the papers describing
+    Fairlearn's mitigation algorithms.
+
+Why not just ignore the sensitive features?
+    Because ignoring the sensitive feature usually doesn't work.
```

Comment:
We shouldn't say "it usually doesn't work" without calling out the use case scenario where it does not work. In this question, there is a hidden assumption about what notion of fairness the user is interested in--most likely the goal is to achieve statistical independence between the sensitive feature and prediction. The answer needs to start by clarifying this notion of fairness and then pointing out that there will be statistical dependence between the sensitive feature and other features, which may will lead to the statistical dependence between the predictor and sensitive features.

Reply:
I was enjoying being blunt :-)

To tell the truth, that hidden assumption hadn't occurred to me. I've made it explicit.